api/grpcserver_integrity_test.go

```diff
@@ -1304,6 +1304,7 @@ func TestGrpcServer_SendActionIntegrity(t *testing.T) {
 		broadcastHandlerCount++
 		return nil
 	}
+	coreService.messageBatcher = nil
```

Comment:
not needed? by default it is `nil`?

Reply:
`messageBatcher` won't be `nil` after it's enabled